src/patternfly/components/AboutModalBox/examples/AboutModalBox.md

```diff
@@ -6,11 +6,10 @@ cssPrefix: pf-c-about-modal-box
 
 ## Examples
 ### Basic
-```hbs isFullscreen
-{{#> about-modal-box about-modal-box--attribute='aria-labelledby="about-modal-title"'}}
+```hbs
```

Comment:
This should probably still be full-screen. There is some wonky parallax action going on with the background image as you scroll the example page.

```suggestion
```hbs isFullscreen
```